### DISCUSSION TITLE
notes/pages/notes/Python/01.%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86

### DISCUSSION THREAD

**Discussion by giscus** [dllcnx/dllcnx.github.io#131] [Announcements]
# notes/pages/notes/Python/01.%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86

基础数据类型 整数(int)：8 浮点数(float)：8.8 字符串(str)：'123',

https://dllcnx.com:18888/notes/pages/notes/Python/01.%E5%9F%BA%E7%A1%80%E7%9F%A5%E8%AF%86.html

**Comment by dllcnx** [comment]
你好
